fearless/Modules/Send/Validators/SendDataValidatingFactory.swift

```diff
@@ -204,7 +211,8 @@ class SendDataValidatingFactory: NSObject {
                 return
             }
             let title = R.string.localizable.commonWarning(preferredLanguages: locale.rLanguages)
-            let message = R.string.localizable.soraBridgeAmountLessFee(preferredLanguages: locale.rLanguages)
+            let originKnownChain = Chain(chainId: originCHainId)?.rawValue ?? ""
```

Comment:
there's better to have some default value instead of empty string. "Uknown" maybe?